src/commands/project/get-display-name.ts

```diff
@@ -0,0 +1,62 @@
+/* eslint-disable @typescript-eslint/no-use-before-define */
+/* eslint-disable @typescript-eslint/camelcase */
+import commander, { CommandOptions } from "commander";
+import * as fs from "fs";
+import { read as readBedrockYaml } from "../../lib/bedrockYaml";
+import { build as buildCmd, exit as exitCmd } from "../../lib/commandBuilder";
+import { logger } from "../../logger";
+import decorator from "./get-display-name.decorator.json";
+
+/**
+ * Executes the command, can all exit function with 0 or 1
+ * when command completed successfully or failed respectively.
+ *
+ * @param opts validated option values
+ * @param exitFn exit function
+ */
+export const execute = async (
+  opts: CommandOptions,
+  exitFn: (status: number) => Promise<void>
+): Promise<void> => {
+  let currentDirectory = process.cwd() + "/bedrock.yaml";
+  try {
+    // First, find a bedrock.yaml file in this directory and if not, iterate by
```

Comment:
Hm. Let's not do this tree climbing. We're pretty strict about running `spk project` and `spk service` commands only in the root of the project repository.

Reply:
@mtarng this command is not meant so much for users to run manually, but just for the pipeline itself, to extract the name and send to introspection storage.  I'm thinking if it should even be baked into spk and available for the user to run as a command or find it confusing in docs on why it's even available, since it's so trivial. @andrebriggs thoughts?